bin/snyk_tests.sh

```diff
@@ -22,13 +24,39 @@ source bin/common/check_returncode.sh
 # authenticate snyk CLI locally
 run_and_check_returncode "snyk auth $SNYK_AUTHENTICATION_TOKEN"
 
-# Iac run limited to 20 runs a month. TBD Discuss with Jon Velapondi.
-# echo "Scanning Infrastructure as code with Snyk for potential vulnerabilities..."
-# snyk iac test || true
-
 # scan orca website
 echo "Scanning open-source code with Snyk for potential vulnerabilities..."
 cd website
 snyk test -d --severity-threshold=high || true
+cd -
+
+# run snyk test for shared libraries
+echo
+echo "Running snyk tests in shared_libraries"
+echo
+cd shared_libraries
+snyk test --severity-threshold=high --file=requirements.txt --package-manager=pip || true
+cd -
+
+function run_snyk_tests() {
+  echo
+  echo "Running snyk test in $1"
+  echo
+
+  cd $1
+  # remove shared library from the requirement files since snyk cannot scan through it
+  grep -v '../../shared_libraries' requirements-dev.txt >> test-requirements-dev.txt
+  grep -v '../../shared_libraries' requirements.txt >> test-requirements.txt
+  snyk test --severity-threshold=high --file=test-requirements-dev.txt --package-manager=pip || true
+  snyk test --severity-threshold=high --file=test-requirements.txt --package-manager=pip || true
+  # remove test files
+  rm test-requirements*.txt
+}
+export -f run_snyk_tests
+
+task_dirs=$(ls -d tasks/*)
```

Comment:
Can we add a couple paths to this? Ideally we'd run this check on `shared_libraries` and `graphql` as well. Don't know if there are any unique challenges there.

Reply:
let me try

Reply:
looks like the existing function will not work for other directories since the folder structure is different. 
Created another function to handle those.